### DISCUSSION TITLE
爱快软路由FTP中文显示乱码

### DISCUSSION THREAD

**Discussion by q965502450** [AlistGo/alist#3031] [Q&A]
https://m.zhiyuekeji.com/2(%E4%B8%AD%E6%96%87%E6%8A%A5%E9%94%99)

![image](https://user-images.githubusercontent.com/41704323/212230228-303e35a6-4508-4812-a7aa-a7b95af30fee.png)

**Comment by NovaShen555** [comment]
反代设置编码格式试试

**Reply by Toby-Shi-cloud** [reply]
nginx的charset似乎只能用于http的server，ftp的能行吗？

**Comment by github-office** [comment]
我把alist的目录开启了ftp，结果电脑打开之后文件全是乱码。ftp上传的文件，alist里面是乱码。
